arch-nspawn.in

```diff
@@ -81,7 +77,7 @@ build_mount_args() {
 
 copy_hostconf () {
 	cp -a /etc/pacman.d/gnupg "$working_dir/etc/pacman.d"
-	echo "Server = $host_mirror" >"$working_dir/etc/pacman.d/mirrorlist"
+	printf 'Server = %s\n' "${host_mirror[@]}" >"$working_dir/etc/pacman.d/mirrorlist"
```

Comment:
This needs to go into a loop to generate several `Server =` entries, no?

Reply:
It works due to the newline in the printf formatter, and the multiple arguments from `host_mirror`.
```
$ foo=(line1 line2 line3)
$ printf '%s\n' "${foo[@]}"
line1
line2
line3
```